engine/src/main/java/org/terasology/rendering/dag/nodes/InitialPostProcessingNode.java

```diff
@@ -53,13 +51,12 @@
  * Stores the result into the InitialPostProcessingNode.INITIAL_POST_FBO, to be used at a later stage.
  */
 public class InitialPostProcessingNode extends AbstractNode {
-    public static final ResourceUrn INITIAL_POST_FBO = new ResourceUrn("engine:fbo.initialPost");
-    public static final ResourceUrn INITIAL_POST_MATERIAL = new ResourceUrn("engine:prog.initialPost");
+    static final ResourceUrn INITIAL_POST_FBO = new ResourceUrn("engine:fbo.initialPost");
```

Comment:
Yeah. Unless I'm mistaken, some other Node makes use of this particular Urn too, so it can't be private - for now. I hope on changing that soon enough :D